lib/heroku/command/pg.rb

```diff
@@ -527,13 +527,15 @@ def links
         dbs = resolver.all_databases.values
       end
 
+      dbs_by_addons = dbs.group_by(&:resource_name)
+
       error("No database attached to this app.") if dbs.compact.empty?
 
-      dbs.each_with_index do |attachment, index|
-        response = hpg_client(attachment).link_list
+      dbs_by_addons.each_with_index do |(resource, attachments), index|
+        response = hpg_client(attachments.first).link_list
         display "\n" if index.nonzero?
 
-        styled_header("#{attachment.display_name} (#{attachment.resource_name})")
+        styled_header("#{attachments.map(&:config_var).join(", ")} (#{resource})")
```

Comment:
Indeed, and I'd like to see that change :speak_no_evil: Perhaps you're right that we shouldn't surprise users, though. The struggle is real. :weary: